kirby/models.py

```diff
@@ -6,8 +6,7 @@
 
 
 class Environment(db.Model):
-    id = db.Column(db.Integer, primary_key=True)
```

Comment:
I'd definitely keep IDs, because it makes it way easier when referencing objects. I agree that on a database level, you can consider that the name **is** the primary key (and it is). 

But when producing URLs, you will be dealing with a lot of nasty stuff when your names have unicode/spaces/special characters in them.

Please keep on with numerical IDs, even if it breaks the normalisation

Reply:
Sure, the model is now using `id` as primary keys and not `name` anymore.  
I still kept the `unique` constraint on the names